src/components/SelectComponent/selectComponent.js

```diff
@@ -1,16 +1,21 @@
 const SelectComponent = (props) => {
-    const { onChange, ariaLabel} = props
-    return(
-     <div className="form-group">
-        <label htmlFor="exampleFormControlSelect1">Select Number</label>
-        <select className="form-control" id="exampleFormControlSelect1" onChange={(e)=>onChange(e)} aria-label={ariaLabel}>
-          <option value="1">1</option>
-          <option value="2">2</option>
-          <option value="3">3</option>
-          <option value="4">4</option>
-          <option value="5">5</option>
-        </select>
-      </div>
-    )
-}
-export default SelectComponent
+  const { onChange, ariaLabel } = props;
+  return (
+    <div className="form-group">
+      <label htmlFor="exampleFormControlSelect1">Select Number</label>
+      <select
+        className="form-control"
+        id="exampleFormControlSelect1"
+        onChange={(e) => onChange(e)}
+        aria-label={ariaLabel}
+      >
+        <option value="1">1</option>
```

Comment:
use array here for options like [{ id: 1, value: 1 }, {id:2, value:2 } ...] and then use map.